<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> A ... Z</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonConv">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Conv</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMostrar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRemover">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover do Arquivo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRemover">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExcluir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAdicionar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar no Arquivo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>377</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConverter_String">
    <property name="title">
     <string>Converter String</string>
    </property>
   </widget>
   <addaction name="menuConverter_String"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
